website/docs/reference/node-selection/syntax.md

```diff
@@ -219,10 +202,13 @@ Another element of job state is the `source_status` of a prior dbt invocation. A
 The following dbt commands produce `sources.json` artifacts whose results can be referenced in subsequent dbt invocations:  
```

Comment:
Being there's only one, can we make this simpler? We could even move it into the previous paragraph. 
```suggestion
The `dbt source freshness` command produces a `sources.json` artifact whose results can be referenced in subsequent dbt invocations. 
```